free-courses-it.md

```diff
@@ -90,6 +91,12 @@
 * [Corso Arduino 2014](https://www.youtube.com/watch?v=WuqS4C4u42E&list=PLA27EZBY5vePO9T6YP3rH8LTTdylz69VE) (POuLpolimi Politecnico di Milano)
 
 
+### Erlang
+
+* [Erlang/OTP](http://www.khaldi.it/art/erlang/erl01/E01.html) - (Jilani KHALDI)
+* [Iniziare con Erlang Language](https://riptutorial.com/it/erlang) - [(riptutorial.com)](https://riptutorial.com)
```

Comment:
no links for platforms

Reply:
Do I have to remove the entry alltogether or just the link at the end?